internal/parsevolume/parsevolume.go

```diff
@@ -1,4 +1,4 @@
-package loader
+package parsevolume
```

Comment:
We should probably move the related types as well, then alias the ones in the old location; this keeps backward-compatibility in case anyone currently imports `cli/compose/loader` and who may be using the things we moved.

Go does allow aliasing types that are internal, which means we can keep the implementation internal, and still keep the existing place. (for now at least). If we'd ever be working on re-implementing, we could split the two (as `command/container` only uses it internally.